src/Http/Controllers/Character/IntelController.php

```diff
@@ -27,7 +27,7 @@
 use Seat\Web\Http\Controllers\Controller;
```

Comment:
# Character Standing Compare
- selecting a standing profile is not calling id-to-name resolver
- sorting on character name is throwing an exception
```
DataTables warning: table id=character-standings-interactions - Exception Message:

	SQLSTATE[42S22]: Column not found: 1054 Unknown column 'characterName' in 'order clause' (SQL: select count(*) as total, `character_affiliations`.`character_id`, `character_affiliations`.`corporation_id`, `character_affiliations`.`alliance_id`, `character_affiliations`.`faction_id`, `standings_profile_standings`.`elementID` as `standing_match_on`, `standings_profile_standings`.`type` as `standing_type`, `standings_profile_standings`.`standing` as `standing` from `character_wallet_journals` left join `character_affiliations` on `character_affiliations`.`character_id` = `character_wallet_journals`.`first_party_id` or `character_affiliations`.`character_id` = `character_wallet_journals`.`second_party_id` inner join `standings_profile_standings` on `standings_profile_standings`.`elementID` = `character_affiliations`.`character_id` or `standings_profile_standings`.`elementID` = `character_affiliations`.`corporation_id` or `standings_profile_standings`.`elementID` = `character_affiliations`.`alliance_id` where `character_wallet_journals`.`character_id` = 90795931 and `standings_profile_standings`.`standings_profile_id` = 1 group by `standings_profile_standings`.`elementID` order by `characterName` asc limit 10 offset 0)
```

- sorting on corporation is throwing an exception
```
DataTables warning: table id=character-standings-interactions - Exception Message:

	SQLSTATE[42S22]: Column not found: 1054 Unknown column 'corporationName' in 'order clause' (SQL: select count(*) as total, `character_affiliations`.`character_id`, `character_affiliations`.`corporation_id`, `character_affiliations`.`alliance_id`, `character_affiliations`.`faction_id`, `standings_profile_standings`.`elementID` as `standing_match_on`, `standings_profile_standings`.`type` as `standing_type`, `standings_profile_standings`.`standing` as `standing` from `character_wallet_journals` left join `character_affiliations` on `character_affiliations`.`character_id` = `character_wallet_journals`.`first_party_id` or `character_affiliations`.`character_id` = `character_wallet_journals`.`second_party_id` inner join `standings_profile_standings` on `standings_profile_standings`.`elementID` = `character_affiliations`.`character_id` or `standings_profile_standings`.`elementID` = `character_affiliations`.`corporation_id` or `standings_profile_standings`.`elementID` = `character_affiliations`.`alliance_id` where `character_wallet_journals`.`character_id` = 90795931 and `standings_profile_standings`.`standings_profile_id` = 1 group by `standings_profile_standings`.`elementID` order by `corporationName` asc limit 10 offset 0)
```

- sorting on alliance is throwing an exception
```
DataTables warning: table id=character-standings-interactions - Exception Message:

	SQLSTATE[42S22]: Column not found: 1054 Unknown column 'allianceName' in 'order clause' (SQL: select count(*) as total, `character_affiliations`.`character_id`, `character_affiliations`.`corporation_id`, `character_affiliations`.`alliance_id`, `character_affiliations`.`faction_id`, `standings_profile_standings`.`elementID` as `standing_match_on`, `standings_profile_standings`.`type` as `standing_type`, `standings_profile_standings`.`standing` as `standing` from `character_wallet_journals` left join `character_affiliations` on `character_affiliations`.`character_id` = `character_wallet_journals`.`first_party_id` or `character_affiliations`.`character_id` = `character_wallet_journals`.`second_party_id` inner join `standings_profile_standings` on `standings_profile_standings`.`elementID` = `character_affiliations`.`character_id` or `standings_profile_standings`.`elementID` = `character_affiliations`.`corporation_id` or `standings_profile_standings`.`elementID` = `character_affiliations`.`alliance_id` where `character_wallet_journals`.`character_id` = 90795931 and `standings_profile_standings`.`standings_profile_id` = 1 group by `standings_profile_standings`.`elementID` order by `allianceName` asc limit 10 offset 0)
```